api/ImageData.json

```
@@ -148,6 +148,41 @@
                 "deprecated": false
               }
             }
+          },
+          "pixelFormat_option": {
+            "__compat": {
+              "description": "`pixelFormat` option",
+              "spec_url": "https://html.spec.whatwg.org/multipage/canvas.html#canvascolortype",
+              "support": {
+                "chrome": {
+                  "version_added": "137"
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false,
+                  "impl_url": "https://bugzil.la/1958830"
+                },
+                "firefox_android": "mirror",
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": false,
+                  "impl_url": "https://webkit.org/b/291196"
+                },
+                "safari_ios": "mirror",
+                "samsunginternet_android": "mirror",
+                "webview_android": "mirror",
+                "webview_ios": "mirror"
+              },
+              "status": {
+                "experimental": false,
```

BCD [considers the feature experimental](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/index.md#setting-experimental):

```suggestion
                "experimental": true,
```